### ISSUE TITLE
Tensorflow 2.0

### ISSUE THREAD

**Issue by michaelrabelo** [bshao001/ChatLearner#84]
Is there any chance of updating the code to be compatible with Tensorflow 2.0?

**Comment by bshao001** [comment]
No. I am not working on or plan to support this project any longer. If you seriously plan to develop a chatbot, I suggest you use Transformer model instead of NMT (this one).

**Comment by michaellin99999** [comment]
if I want to change it from NMT to transformer model, which script should I start with first?
